src/+otp/+vanderpol/+presets/ExtendedPrecision.m

```diff
@@ -0,0 +1,29 @@
+classdef ExtendedPrecision < otp.vanderpol.VanderpolProblem
+    methods
+        function obj = ExtendedPrecision(epsilon)
+            if nargin < 1
+                epsilon = sym(10^-6);
+            end
+            
+            tspan = ([0, 0.5]);
+            z0 = (epsilon.^(11:-1:0))*[sym(1499571590693390159668)/16677181699666569, ...
```

Comment:
This is still converted to double first.  You need to use use string input like `sym('1499571590693390159668/16677181699666569')` to avoid losing digits.

Reply:
In fact, if we want to actually use vpa, then we would want to define the precision, then do this

```
vpa(sym('1499571590693390159668/16677181699666569')/sym('16677181699666569'), precision)
```

where `precision` is the number of digits we want to keep (have it be an input to the preset?)

Reply:
It's simpler to do `vpa('1499571590693390159668/16677181699666569')`

Reply:
from my understanding, chcracter vector calls to vpa are discouraged. I think rationals are fine though. hmmm